docs/changelog.rst

```diff
@@ -19,6 +19,10 @@ Bug fix release.
 
 - MPD: Notify idling clients when a seek is performed. (Fixes: :issue:`1331`)
 
+- Core: Fix :meth:`~mopidy.core.PlaybackController._change_track` to mark
```

Comment:
I don't think the method name is relevant for the changelog, as it's a private method.